### PR TITLE
Fix grub configuration file permissions on Debian

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request fixes a bug where the `grub` configuration file permissions are not set correctly on Debian systems.

## 💭 Motivation and context ##

Resolves cisagov/cool-system#79.

## 🧪 Testing ##

I built a new OpenVPN staging AMI with these changes and verified that the `grub` configuration file permissions were correct.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).

### CODE REVIEW

**Comment by jsf9k** [comment]
The failing Travis builds are a red herring.  They are from the repo from which we forked.  We don't have tests for this repo.

**Review by hillaryj — APPROVED** [review]
🥇

File: tasks/fix_grub_config_perms.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+# It would be great to put this whole mess in a block and let that
+# block's become: yes and tags: grub apply to all these tasks.
+# Unfortunately that does not appear to work.  When I try that I get
+# error messages from Ansible that look like "Unexpected exception:
+# Block object has no attribute get_name".
```
Comment (dav3r) [inline tasks/fix_grub_config_perms.yml:6]: image

**Review by dav3r — APPROVED** [review]
This looks great - nice job! 🤜 🐞

**Review by felddy — APPROVED** [review]
🪱

**Pull request merged**